skeleton/controller/admin_controller.php.twig

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ *
+ * {{ EXTENSION.extension_display_name }}. An extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) {{ "now"|date("Y") ~ (AUTHORS.0.author_name ? ', ' ~ AUTHORS.0.author_name) ~ (AUTHORS.0.author_homepage ? ', ' ~ AUTHORS.0.author_homepage) }}
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+namespace {{ EXTENSION.vendor_name }}\{{ EXTENSION.extension_name }}\controller;
+
+/**
+ * {{ EXTENSION.extension_display_name }} admin controller.
+ */
+class admin_controller
```

Comment:
Let's call this `app_controller` so we keep this and the other module controllers all consistent and generic.